### ISSUE TITLE
parallel pynast requires command line BLAST

### ISSUE THREAD

**Issue by gregcaporaso** [biocore/qiime#1989]
This shouldn't be the case, it means that the base install does not support the parallel otu picking workflows. Working on this now. 

```bash
$ which formatdb
/Applications/blast-2.2.26/bin/formatdb
(qiime)08:18:12 otus$ parallel_align_seqs_pynast.py -i rep_set//seqs_rep_set.fasta -o pyn_par
(qiime)08:18:43 otus$ # job completes 

## remove formatdb from $PATH
(qiime)08:19:44 otus$ which formatdb
(qiime)08:19:47 otus$ parallel_align_seqs_pynast.py -i rep_set//seqs_rep_set.fasta -o pyn_par
Traceback (most recent call last):
  File "/Users/caporaso/.virtualenvs/qiime/bin/parallel_align_seqs_pynast.py", line 10, in <module>
    execfile(__file__)
  File "/Users/caporaso/Dropbox/code/Qiime/scripts/parallel_align_seqs_pynast.py", line 104, in <module>
    main()
  File "/Users/caporaso/Dropbox/code/Qiime/scripts/parallel_align_seqs_pynast.py", line 100, in main
    suppress_submit_jobs=opts.suppress_submit_jobs)
  File "/Users/caporaso/Dropbox/code/Qiime/qiime/parallel/util.py", line 133, in __call__
    self._precommand_initiation(input_fp, output_dir, working_dir, params)
  File "/Users/caporaso/Dropbox/code/Qiime/qiime/parallel/align_seqs.py", line 32, in _precommand_initiation
    output_dir=get_qiime_temp_dir())
  File "/Users/caporaso/.virtualenvs/qiime/lib/python2.7/site-packages/bfillings/formatdb.py", line 116, in build_blast_db_from_fasta_path
    fdb = FormatDb(WorkingDir=output_dir, HALT_EXEC=HALT_EXEC)
  File "/Users/caporaso/.virtualenvs/qiime/lib/python2.7/site-packages/burrito/util.py", line 202, in __init__
    self._error_on_missing_application(params)
  File "/Users/caporaso/.virtualenvs/qiime/lib/python2.7/site-packages/burrito/util.py", line 467, in _error_on_missing_application
    "Is it in your path?" % command)
burrito.util.ApplicationNotFoundError: Cannot find formatdb. Is it installed? Is it in your path?

## serial PyNAST works without formatdb
(qiime)08:19:52 otus$ align_seqs.py -i rep_set//seqs_rep_set.fasta -o pyn_ser
(qiime)08:20:26 otus$ # job completes
```

**Comment by antgonza** [comment]
But I think this is a burrito issue, no?

**Comment by gregcaporaso** [comment]
I don't think so. Why do you think that?

**Comment by antgonza** [comment]
For who is raising the error:
```
  File "/Users/caporaso/.virtualenvs/qiime/lib/python2.7/site-packages/bfillings/formatdb.py", line 116, in build_blast_db_from_fasta_path
    fdb = FormatDb(WorkingDir=output_dir, HALT_EXEC=HALT_EXEC)
  File "/Users/caporaso/.virtualenvs/qiime/lib/python2.7/site-packages/burrito/util.py", line 202, in __init__
    self._error_on_missing_application(params)
  File "/Users/caporaso/.virtualenvs/qiime/lib/python2.7/site-packages/burrito/util.py", line 467, in _error_on_missing_application
    "Is it in your path?" % command)
burrito.util.ApplicationNotFoundError: Cannot find formatdb. Is it installed? Is it in your path?
```

**Comment by gregcaporaso** [comment]
That's the expected behavior from burrito. If ``formatdb`` is not available, then calling the application controller that it wraps raises an ``ApplicationNotFound`` error. 

I have the fix in progress, and we'll put it in 1.9.1 (planned for Friday). Basically, the parallel wrapper was still formatting a BLAST database that isn't being used (earlier versions of PyNAST did rely on BLAST for the sequence searching step).

**Comment by antgonza** [comment]
Got it, thanks.
